app/src/main/java/com/tvtracker/ui/main/BrowseViewModel.kt

```diff
@@ -148,7 +139,7 @@ class BrowseViewModel(var mediaService: IMediaService = MediaService()): ViewMod
                 selectedMediaItem.id = document.id
                 val handle = document.set(selectedMediaItem)
                 handle.addOnSuccessListener { Log.d("Firebase", "Document Saved") }
-                handle.addOnFailureListener { Log.e("Firebase", "Error: $it") }
+                handle.addOnFailureListener { Log.e("Firebase", "Error: $user") }
```

Comment:
In lambdas that are short and not nested, it's recommended to use its convention instead of declaring the parameter explicitly. In nested lambdas with parameters, always declare parameters explicitly. Declare user because that is what is being handled. 
https://kotlinlang.org/docs/coding-conventions.html#lambda-parameters

Reply:
These are two different objects.
it == the error message.
user == a user.

So, this changes functionality a bit.  I'd keep it as it were, as the error message is the most important thing to log here.